src/passes/blake3_template

```diff
@@ -0,0 +1,384 @@
+//! This is the reference implementation of BLAKE3. It is used for testing and
```

Comment:
So that rust-analyzer would leave me alone about this file not being included in the module tree :D Also I guess it's to make sure that people don't think we compile it? I'm fine with adding the Rust extension if you think it's best